src/include/btmem.h

```diff
@@ -899,6 +900,7 @@ struct __wt_update {
 #define	WT_UPDATE_MODIFIED	1	/* partial-update modify value */
 #define	WT_UPDATE_RESERVED	2	/* reserved */
 #define	WT_UPDATE_STANDARD	3	/* complete value */
+#define	WT_UPDATE_IMPOSSIBLE	255
```

Comment:
> Could you please have a look at restructuring the code so this isn't necessary?

I tried to refactor `__wt_col_modify` and `__wt_row_modify` a couple of weeks ago, and didn't get anywhere better.

> If there is no way around it, a better name might be `WT_UPDATE_INVALID`, but it still feels wrong to define a value here that can never appear in the struct.

Agreed.

In 09c2012, I changed `WT_UPDATE.type` to an `int8_t` so I can pass (-1) as the out-of-band value. That's a more traditional approach, and avoids `WT_UPDATE_IMPOSSIBLE`.